### DISCUSSION TITLE
Exclude Dependencies from Vulnerability Alerts

### DISCUSSION THREAD

**Discussion by karlderkaefer** [renovatebot/renovate#23732] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I have following package rule for my typescript project:
```json
    {
      "groupName": "cdk monorepo",
      "matchPackagePatterns": [
        "^@aws-cdk",
        "^aws-cdk",
        "^constructs"
      ],
      "addLabels": ["cdk-core"],
      "semanticCommitType": "chore"
    }
```
So I would expect that dependency `aws-cdk-lib` is updated within the monorepo PR. However it turn out it is not. It seems the vulnerabilityAlerts get higher priority and my custom package rule is ignored.

```
DEBUG: GitHub vulnerability details
{
  "alerts": {
    "npm/http-cache-semantics": {
      "< 4.1.1": "4.1.1"
    },
    "npm/aws-cdk-lib": {
      ">= 2.0.0, < 2.80.0": "2.80.0"
    },
    "npm/semver": {
      ">= 7.0.0, < 7.5.2": "7.5.2",
      "< 5.7.2": "5.7.2"
    }
  }
}
..
DEBUG: alert package rules
{
  "alertPackageRules": [
  {
      "matchDatasources": [
        "npm"
      ],
      "matchPackageNames": [
        "aws-cdk-lib"
      ],
      "matchCurrentVersion": "= 2.76.0",
      "matchFileNames": [
        "package-lock.json"
      ],
      "allowedVersions": "2.80.0",
```
Actually I want to ignore vulnerabilityAlerts for a specific dependency, however the documentation does not really say which fields are supported. 

### What you tried so far.

I tried to exclude package name without success
```
  "vulnerabilityAlerts": {
    "automerge": true,
    "excludePackageNames": ["aws-cdk-lib"]
  }
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
You can dismiss individual alerts for dependencies through the GitHub UI. I don't think there's any way to do it with package rules
